### ISSUE TITLE
`ariaLabel: true` does not work for setting button's `ariaLabel`

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#236248]
For a button, set `ariaLabel: true`. Focus the button with a screen reader. Note that you don't hear the button's title.

Setting `ariaLabel` on the button directly works.

_Originally posted by @meganrogge in https://github.com/microsoft/vscode/pull/236096#discussion_r1887160275_

**Comment by bpasero** [comment]
Feel free to make changes as you see fit and I can review 🙏

**Comment by meganrogge** [comment]
Just discovered this doesn't work when the `label` is not set because that's where this gets applied

https://github.com/microsoft/vscode/blob/e3d033cf31ffba88c2185b1f79f6e57caea39fa3/src/vs/base/browser/ui/button/button.ts#L264-L269

**Comment by meganrogge** [comment]
We should probably set the `ariaLabel` also when `icon` is set.

**Comment by meganrogge** [comment]
To verify, add a button and don't set `label`. Also, don't set `ariaLabel`. Set `title`. The `ariaLabel` should be set to `title`.
